release-notes/8.0/supported-os.md

```diff
@@ -102,7 +102,7 @@ OS                            | Version                 | Architectures     |
 [iOS Simulator][iOS]          | 11.0+                   | x64, Arm64        |
```

Comment:
iOS 11 stopped receiving updates years ago and it is no longer supported by Appple. The oldest iOS that is still receiving security updates is iOS 12.